### ISSUE TITLE
Device Scanning Issue in iOS Demo Project

### ISSUE THREAD

**Issue by atikomtancharoen** [katoemba/SwiftUPnP#11]
Hi,

Thank you for sharing this library.

I tried using the library in my iOS Demo Project to scan for devices like Android TV or Tizen OS, but there was no response at all.

I'm not sure if you provided an example project for testing, in case I implemented something incorrectly.

**Comment by katoemba** [comment]
You need to enable multicast, see the readme for details.

Additionally, by default only the following device signatures are picked up: 
    urn:schemas-upnp-org:device:MediaServer:1
    urn:linn-co-uk:device:Source:1
    urn:av-openhome-org:device:Source:1

You can pass other device signatures as parameter to startDiscovery().

**Comment by atikomtancharoen** [comment]
Thank you for your response.

Do you have any recommendations or documents for me to read about multicast setup?

**Comment by katoemba** [comment]
No, apart from the link in the read-me I have no further references.
